src/daal4py.cpp

```diff
@@ -851,6 +851,46 @@ void c_train_test_split(data_or_file & orig, data_or_file & train, data_or_file
     daal::data_management::internal::trainTestSplit<int>(origTable, trainTable, testTable, trainIdxTable, testIdxTable);
 }
 
+double c_roc_auc_score(data_or_file & y_true, data_or_file & y_test, char dtype)
+{
+    const int col_true = y_true.table->getNumberOfColumns();
+    const int row_true = y_true.table->getNumberOfRows();
+    const int col_test = y_test.table->getNumberOfColumns();
+    const int row_test = y_test.table->getNumberOfRows();
+    if (row_true != 1 || row_test != 1 || col_true != col_test) {
+        PyErr_SetString(PyExc_RuntimeError, "Unknown shape data");
+        return NULL;
+    }
+
+    auto table_true = get_table(y_true);
+    auto table_test = get_table(y_test);
+
+    switch (dtype)
```

Comment:
Use switch similar from [here](https://github.com/IntelPython/daal4py/blob/master/src/daal4py.cpp#L214)

You don't have to pass 0 and 1 and it's so awful to define the type :)